report/sections/Chapter 2/architecture.tex

```diff
@@ -0,0 +1,51 @@
+\section{Architecture}
+To accomodate a platform that supports the issues described in Section xxxx we have chosen a client-server architecture. 
+In this architecture communication betwen the client and server is implemented using the HTTP protocol. 
+The front-end component is responsible for the UI for the platform and must provide the users with described as use-cases.
+The back-end of the system supports the heavy lifting for the front end system, utilizing computer power of a server rather than the client's browser. 
```

Comment:
```suggestion
Meanwhile, the back-end of the system supports the heavy lifting of the front-end system, utilizing the computing power of a server rather than the client's browser. 
```